styles.css

```diff
@@ -0,0 +1,75 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+body {
+  min-height: 100%;
+  margin: 0;
+  font-family: Helvetica;
+  font-size: 18px;
+}
+.post__header__logo {
+  height: 80px;
+  margin: 0 20px;
+  border-bottom: 2px solid #fff;
+}
+.post__header {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  width: 50%;
+  background-image: url('./images/hunkahunkakittylove.jpg');
+  background-size: cover;
+  background-position: 50% 50%;
+  background-color: ##DCBF79;
+  color: #FFF;
+}
+.post__headerNav {
+  list-style: none;
+  margin-top: 15px;
+  padding-left: 20px;
+}
+.post__headerNav > li {
+  display: inline-block;
+}
+.post__headerNav > li:after {
+  content: "-";
+}
+.post__headerNav > li:last-of-type:after {
+  content: none;
+}
+.post__headerNav > li > a {
+  color: inherit;
+  text-decoration: none;
+  font-size: .8em;
+}
+.post__headerNav > li > a:hover {
+  background-color: #D6AC30
```

Comment:
try to remember the `;` at the end of each line, even if it's the last/only line--it's good practice